### ISSUE TITLE
Feature request : input by REST/GRAPHQL API 

### ISSUE THREAD

**Issue by lucasverra** [ananas-analytics/ananas-desktop#71]
as seen [in HN](https://news.ycombinator.com/item?id=20499114) :

> Can we hook this to an api GET request ? I guess i could API -> download JSON -> Ananas, but you know..:)

🙏🏼

Cheers !

**Comment by bhou** [comment]
possible step config:
```yaml
config:
    method: GET | POST | PUT | DELETE
    contentType: json | text # auto inject content type header with this
    headers:
        key: value
        ...
    body: |
        [request body here]
    plainText: false # true to ignore content type, 
                     # see the result as plain text, useful when combining with
                     # JS/Python steps for complex return structures
    jsonFilter: data.users # only useful for json, support jq syntax?
    lineDelimiter: \n # only useful for text, result as multiple line
```

The first step could be supporting only `application/json` and `application/text` content type. 

If the result is in JSON, need to format the JSON to row-based (each line a JSON record) as a JSON log

**Comment by bhou** [comment]
Depends on #91

**Comment by bhou** [comment]
included in release 0.9.0
